packages/ethereum-storage/src/ethereum-storage-ethers.ts

```diff
@@ -75,7 +75,7 @@ export class EthereumStorageEthers implements StorageTypes.IStorageWrite {
     this.logger.debug(`TX ${tx.hash} submitted, waiting for confirmation...`);
 
     void tx
-      .wait()
+      .wait(1)
```

Comment:
this is the default value

https://github.com/ethers-io/ethers.js/blob/ba967875f71019d367606f2b0bfae27879723a57/src.ts/providers/provider.ts#L822

Reply:
I think I'm off-by-one but that the concept is still valid. I'll increase it to 2 confirmations because Gnosis Chain frequently experiences 1 block reorgs.

Reply:
can you make it configurable by env var?

Reply:
Good idea. The reorg frequency differs between chains.